lib/expansion_rules.rb

```diff
@@ -128,7 +128,7 @@ def details_fields(*fields)
 
   POSSIBLE_FIELDS_FOR_LINK_EXPANSION = DEFAULT_FIELDS +
     %i[details] +
-    %i[id state phase description unpublishings.type] -
+    %i[id state phase description auth_bypass_ids unpublishings.type] -
```

Comment:
Looks like this slipped in by accident in commit number two?

Reply:
Thanks - good spot.